src/Database/Attach/File.php

```diff
@@ -144,6 +144,28 @@ public function fromData($data, $filename)
         return $file;
     }
 
+    /**
+     * Creates a file object from url
+     * @param $url string Filename
+     * @return $this
+     */
+    public function fromLink($url)
```

Comment:
method should be `fromUrl`, also `Http` needs to be imported at the top of the file.